scripts/genvimvim.lua

```diff
@@ -115,7 +117,7 @@ end
 local nvimau_start = 'syn keyword nvimAutoEvent contained '
 w('\n\n' .. nvimau_start)
 
-for au, _ in pairs(auevents.nvim_specific) do
+for au, _ in vim.spairs(auevents.nvim_specific) do
```

Comment:
technically this is no longer needed , right (or we could revert the changes to auevents.lua)?

Though `vim.spairs()` still seems like something we generally want.

Reply:
Ah, yes, I meant to revert these changes in auevents.lua, good one. Thanks, updated.